packages/iov-ethereum/src/ethereumconnection.spec.ts

```diff
@@ -215,22 +214,71 @@ describe("EthereumConnection", () => {
     });
   });
 
-  it("can get nonce", async () => {
-    pendingWithoutEthereum();
-    const connection = await EthereumConnection.establish(testConfig.base);
+  describe("getNonce", () => {
+    it("can get nonce", async () => {
+      pendingWithoutEthereum();
+      const connection = await EthereumConnection.establish(testConfig.base);
 
-    // by address
-    {
-      const nonce = await connection.getNonce({ address: testConfig.address as Address });
-      expect(nonce).toEqual(testConfig.nonce);
-    }
-
-    // by pubkey
-    {
-      const nonce = await connection.getNonce({ pubkey: testConfig.pubkey });
-      expect(nonce).toEqual(testConfig.nonce);
-    }
-    connection.disconnect();
+      // by address
+      {
+        const nonce = await connection.getNonce({ address: testConfig.address });
+        expect(nonce).toEqual(testConfig.nonce);
+      }
+
+      // by pubkey
+      {
+        const nonce = await connection.getNonce({ pubkey: testConfig.pubkey });
+        expect(nonce).toEqual(testConfig.nonce);
+      }
+      connection.disconnect();
+    });
+  });
+
+  describe("getNonces", () => {
+    it("can get 0/1/2 nonces", async () => {
+      pendingWithoutEthereum();
+      const connection = await EthereumConnection.establish(testConfig.base);
+
+      // by address, 0 nonces
+      {
+        const nonces = await connection.getNonces({ address: testConfig.address }, 0);
+        expect(nonces.length).toEqual(0);
+      }
+
+      // by address, 1 nonces
+      {
+        const nonces = await connection.getNonces({ address: testConfig.address }, 1);
+        expect(nonces.length).toEqual(1);
+      }
+
+      // by address, 2 nonces
+      {
+        const nonces = await connection.getNonces({ address: testConfig.address }, 2);
+        expect(nonces.length).toEqual(2);
+        expect(nonces[1].toNumber()).toEqual(nonces[0].toNumber() + 1);
```

Comment:
How about checking like above, not just difference...
eg. 
```ts
expect(nonces[0]).toEqual(testConfig.nonce)
expect(nonces[1].toNumber()).toEqual(testConfig.nonce,toNumber() + 1)
```

Reply:
done